theme-release-110.rst

```diff
@@ -60,6 +60,8 @@ If you have a project on Read the Docs without a Python requirements file ``requ
 You can read more about adding a ``requirements.txt`` in our :doc:`Documentation about Reproducible Builds <readthedocs:guides/reproducible-builds>`.
 
 
+.. _sphinx_rtd_theme110_upcoming_releases:
```

Comment:
I think this is not required. We can use `autosectionlabel` for this.

Reply:
Sure, I'll activate that instead :+1:

Reply:
```
Warning, treated as error:
code/readthedocs-blog/ads-with-uniregistry.rst:69:duplicate label get started, other instance in code/readthedocs-blog/adding-markdown-support.rst
make: *** [Makefile:53: html] Error 2
```

Maybe we should add this extension after #188 ?

Reply:
Resolving for now, let's get back to this at the next cross-reference :+1: